src-tauri/src/main.rs

```diff
@@ -63,6 +68,11 @@ fn main() {
                 mesh::device::MeshGraph::new(),
             ))),
         })
+        .manage(ActiveMeshState {
+            inner: Arc::new(async_runtime::Mutex::new(Some(
+                analytics::state::State::new(HashMap::new(), false),
```

Comment:
We're currently passing in the bool `save=false` to our timeline, so we can't get repeated snapshots. I have a couple of questions: 
1. I'm not sure why we're initializing a new State struct here and on line 109 - can we move it somewhere else? 
2. I'm not sure if we should enable save by default (it feels a bit messy since we haven't given the user the ability to specify a save directory or any direct control over the process). Should we pass the save flag in from the frontend?

Reply:
What do you mean by "repeated snapshots"?

Reply:
Good call, we should init state to `None` if it hasn't been initialized in this case.

Reply:
@barkincavdaroglu whatever we're saving if we pass in `save=true`

Reply:
When save is true we just write timeline (which should already be in memory) to external space. No need to save data for MVP / technigala

Reply:
Resolving conversation by defaulting to `save` to `false` and initializing data structures in main as `None`.